helm/cluster-azure/README.md

```diff
@@ -0,0 +1,112 @@
+| `Cluster configuration.nodePools[*].customNodeTaints[*]` | **Node taint**|**Type:** `object`<br/>|
+| `Cluster configuration.nodePools[*].customNodeTaints[*].effect` | **Effect**|**Type:** `string`<br/>|
+| `Cluster configuration.nodePools[*].customNodeTaints[*].key` | **Key**|**Type:** `string`<br/>|
+| `Cluster configuration.nodePools[*].customNodeTaints[*].value` | **Value**|**Type:** `string`<br/>|
+| `Cluster configuration.nodePools[*].disableHealthCheck` | **Disable HealthChecks for the MachineDeployment**|**Type:** `boolean`<br/>|
+| `Cluster configuration.nodePools[*].failureDomain` | **Select zone where to deploy the nodePool**|**Type:** `string`<br/>|
+| `Cluster configuration.nodePools[*].instanceType` | **VM size**|**Type:** `string`<br/>|
+| `Cluster configuration.nodePools[*].name` | **Name** - Unique identifier, cannot be changed after creation.|**Type:** `string`<br/>|
+| `Cluster configuration.nodePools[*].replicas` | **Number of nodes**|**Type:** `integer`<br/>|
+| `Cluster configuration.nodePools[*].rootVolumeSizeGB` | **Root volume size (GB)**|**Type:** `integer`<br/>|
+| `Cluster configuration.provider` | **Cluster API provider name**|**Type:** `string`<br/>|
+| `Cluster configuration.providerSpecific` | **Azure settings**|**Type:** `object`<br/>|
+| `Cluster configuration.providerSpecific.azureClusterIdentity` | **Identity** - AzureClusterIdentity resource to use for this cluster.|**Type:** `object`<br/>|
+| `Cluster configuration.providerSpecific.azureClusterIdentity.name` | **Name**|**Type:** `string`<br/>**Default:** `"cluster-identity"`|
+| `Cluster configuration.providerSpecific.azureClusterIdentity.namespace` | **Namespace**|**Type:** `string`<br/>**Default:** `"org-giantswarm"`|
+| `Cluster configuration.providerSpecific.location` | **Location**|**Type:** `string`<br/>**Default:** `"westeurope"`|
+| `Cluster configuration.providerSpecific.network` | **Azure network settings** - Azure VNet peering and other Azure-specific network settings.|**Type:** `object`<br/>|
+| `Cluster configuration.providerSpecific.network.peerings` | **VNet peerings** - Specifying VNets (their resource groups and names) to which the peering is established.|**Type:** `array`<br/>**Default:** `[]`|
+| `Cluster configuration.providerSpecific.network.peerings[*]` | **VNet peering**|**Type:** `object`<br/>|
+| `Cluster configuration.providerSpecific.network.peerings[*].remoteVnetName` | **VNet name** - Name of the remote VNet to which the peering is established.|**Type:** `string`<br/>**Value pattern:** `^[-\w\._]+$`<br/>|
+| `Cluster configuration.providerSpecific.network.peerings[*].resourceGroup` | **Resource group name** - Resource group for the remote VNet to which the peering is established.|**Type:** `string`<br/>**Value pattern:** `^[-\w\._\(\)]+$`<br/>|
+| `Cluster configuration.providerSpecific.subscriptionId` | **Subscription ID**|**Type:** `string`<br/>**Default:** `"PLACEHOLDER"`|
```

Comment:
i have been planning to fix this one ( which is causing some clusters created through happa to get stuck )

```
➜ k get cluster -n org-giantswarm
NAME     PHASE         AGE     VERSION
glippy   Provisioned   82d     
mx4c3    Deleting      4d18h   
yz48c    Deleting      2d22h  
```

this should  be a required value with on default

Reply:
Totally fine to do that. We can always update the docs using

    schemadocs generate helm/cluster-azure/values.schema.json -o helm/cluster-azure/README.md